### PR TITLE
fixed: language filter

### LINKED ISSUE #126: ✨ Fix language filter
## Description
I've just tidied up this project but I seem to have broken the language filter thing in the process. Would anyone mind going ahead and fixing it? (using Tailwind and Vue)

### PR DESCRIPTION
**Pull request by wooojek**
# Description
`index.html`

- Added `@click` to filter toggle button
- Fixed `@click` callback for language buttons
- Added `{{language}}` to display correct language as button text

`app.js`

- Fixed template string in fetch (missing backticks ``)

# Screenshots
If applicable, please leave some screenshots showing the changes you've made.

# Issues
> Closes #126

### CODE REVIEW

**Comment by wooojek** [comment]
@damcclean

**Comment by damcclean** [comment]
Hey thanks for your contribution! Exactly what I was looking for.

Although, I've got a few suggestions:

* Adding a background to the list of languages
* Having the element on top of the background (not sure if that's the best way of describing what I mean) and just under the filter button.

**Comment by wooojek** [comment]
@damcclean can you tell me more about second bullet point?
about first one, check the screenshot
Screenshot 2019-10-02 at 20 29 51

File: index.html [hunk 1]
```diff
@@ -48,8 +48,9 @@ <h1 class="text-5xl font-bold text-primary uppercase">
             v-for="language in languages"
             class="font-thin text-primary text-1xl m-2 language"
             :text="language"
-            @click="chooseLanguage(language)"
+            @click="() => chooseLanguage(language)"
           >
+          {{language}}
```
Comment (damcclean) [inline index.html]: Just for consistency, would you mind adding spacing between `language` and the curly brackets 😄

File: index.html [hunk 2]
```diff
@@ -48,8 +48,9 @@ <h1 class="text-5xl font-bold text-primary uppercase">
             v-for="language in languages"
             class="font-thin text-primary text-1xl m-2 language"
             :text="language"
-            @click="chooseLanguage(language)"
+            @click="() => chooseLanguage(language)"
```
Comment (damcclean) [inline index.html]: why do you need the `() => ` part?

File: index.html (hunk 1 above)
Comment (wooojek) [reply index.html]: done :)

File: index.html (hunk 2 above)
Comment (wooojek) [reply index.html]: you don't, got used to react way of calling event handlers :)

Comment (wooojek) [reply index.html]: fixed it

**Comment by damcclean** [comment]
> @damcclean can you tell me more about second bullet point?
> about first one, check the screenshot
> Screenshot 2019-10-02 at 20 30 00

1st one, that looks fine, maybe just add `rounded-lg` to as a class so we can get a cornered background.

Yeah, don't really know how to describe the second one. Kind of like a dropdown. 

Take the Github dropdown for example:
image
It's straight under where the profile picture is and it's hovering on top of other elements in the page.

Hopefully that makes more sense 😂

**Comment by damcclean** [comment]
Awesome! Thanks for your work on this one!

**Comment by wooojek** [comment]
@damcclean thanks!

**Pull request merged**